recipe/meta.yaml

```diff
@@ -67,6 +67,7 @@ requirements:
     - python
     - setuptools
     - {{ mpi }}
+    - openssh
```

Comment:
I do not get it, the only thing that changed is adding `openssh` ? How is this related to `ML-PACE` ?

Reply:
I added openssh to fix some other error, which I cant find now since the logs expired. I will remove it and see if there are any issues.

Reply:
These changes are already included in the current `main` branch https://github.com/conda-forge/lammps-feedstock/blob/main/recipe/build.sh

Reply:
I am sure I follow, it seems that ML-PACE is turned off currently in the main branch. Or am I missing something?

Reply:
Looking at the files in this pull request, you do not change the `build.sh` script just the `run.py` and the `meta.yaml` files are changed. So it seems this pull request does not enable the `ML-PACE` package either.

Reply:
Ok, thanks, I see the point now. Shall I simply close this and open a new pull request with ML-PACE on and we see if everything works?